docs/ruleset_functions/get_zones_computer_rooms.md

```diff
@@ -0,0 +1,24 @@
+# get_zones_computer_rooms  
+
+**Description:** Get the list of zones that have at least one computer room space associated with them in the P-RMR.  
+
+**Inputs:**  
+- **P-RMR**: To determine if any of the zones have computer rooms associated with them in the P-RMR.  
+
+**Returns:**  
+- **zones_with_computer_room_list_p**: A list that saves all zones that have at least one computer room associated with them.  
+ 
```

Comment:
The associated RDS is for systems that primarily serve computer rooms. I suggest assuming that "primarily"= computer rooms accounting for more than 50% of floor area of spaces served by the HVAC systems. So I would update this function accordingly, or just implement the criteria within the RDS without using a function.

Reply:
@mkarpman here we're simply checking if a zone includes spaces of type computer room. So maybe this function should return 2 values- List of zones with space types = computer room and Area of space type = computer room for each zone.
The calculation of area served by HVAC system would be done within the function, correct?

Reply:
Implemented a hybrid of your two comments. The function will return the following:
Returns a dictionary with the zones that have at least one computer room space associated with them in the P-RMR as the keys. The values associated with each key are in a list form. The list associated with each key contains the computer room floor area as the first item in the list and the total zone floor area as the second item in the list.